docs/docs-ref-autogen/excel/excelscript/excelscript.shape.yml

```diff
@@ -196,7 +196,34 @@ methods:
       content: 'getImage(): Image;'
       return:
         type: '<xref uid="ExcelScript!ExcelScript.Image:interface" />'
-        description: ''
+        description: |-
+
+
+          #### Examples
+
+          ```javascript
```

Comment:
Seems like this should say TypeScript

Reply:
Looks like they all do that. Fun!

Reply:
Tracing with [this internal item](https://office.visualstudio.com/OC/_workitems/edit/5088272).